### ISSUE TITLE
Run without access to multiplayer planes (might allow usage alongside other mp clients)

### ISSUE THREAD

**Issue by TwinFan** [TwinFan/LiveTraffic#45]
**Current Situation / Problem**
Only one multiplayer client can actively run at a time. Other clients like XSquawkBox, X-IvAp, or PilotEdge take control of multiplayer aircrafts and never release it. That makes it difficult to have such clients run in parallel to each other or to LiveTraffic. This behaviour requires users to move plugins in and out of the plugins folder. Issue #39 eases live a bit by having LT delay acquiring multiplayer planes, but still requires other MP clients to be off.

**Suggested Solution**
Multiplayer planes are required for the TCAS hack only. So it might be possible to offer an option not to use multiplay planes, i.e. dispense with the TCAS functionality, but still use LT for drawing 3D planes.

This will require quite some changes to the xplanemp library.

**Alternatives**
Move plugins in and out of the plugin folder...

**Benefits**
Parallel usage of LiveTraffic and at least one other multiplayer client.

**Comment by TwinFan** [comment]
Update documentation, specifically: Limitations & Installation, to cover limitations of running multiplayer clients in parallel, and the need to configure MP planes for TCAS (moved from requirements)

**Comment by 172MC** [comment]
If both PilotEdge and LT are in the Plugins folder (LT not set to display aircraft), it seems that LT is not "playing nice" and PilotEdge  does not show traffic (I was online on PE and heard another pilot say that I taxied thought his plane, which I had not seen). Could it be because LT loads its CSLs and PilotEdge uses a completely different set of CSLs?

[Log.txt](https://github.com/TwinFan/LiveTraffic/files/2770032/Log.txt)

**Comment by 172MC** [comment]
Deleted posts about LT not playing nice with PilotEdge: it was my mistake, I had forgotten to disable LT in the Plugins drop-down menu

**Comment by TwinFan** [comment]
But I had the complete text in the notification, including the attached `Log.txt` and had a look anyway ;)

As said at other places: It shall not be necessary to disable LT in the Plugin Admin.
I am not a PilotEdge user and can't exactly say what happened but from the Log is clear: LT did not request any network data and did not display a single aircraft.

What may happen is that X-Plane displays AI aircrafts. All of us usually configure a number of AI aircrafts for LT (and other multiplayer clients) to be able to display TCAS blibs. LT has not requested control of multiplayer planes, so X-Plane has control and probably uses that. I can't find final proof in the `Log.txt` either, though. But I see some I/ATC and AUTOGEN activity which usually is logged around AI plane usage.

**Comment by 172MC** [comment]
Thank you. As you read in the log, I have no AI planes enabled. The I/ATC lines are probably related to the fact that the PTT button for PilotEdge (PE) is mapped to the XP contact_atc command.

I tested PE with LT in the plugins folder with and without disabling LT in the Plugins drop-down Enable/Disable window; in both cases, I can connect to PE, and everything seems to work except the multiplayer aircraft, which DO show if disable LT before connecting to PE and DO NOT show if I do not disable LT first and connect to PE.

**Comment by 172MC** [comment]
You are right, today, I again forgot to disable LT and flew on PilotEdge but this time the other (multiplayer) PE aircraft were visible. Not sure why it did not work yesterday ;-)
